source/sonata_simulation.rst

```diff
@@ -96,6 +97,27 @@ example::
        "integration_method" : 2
   }
 
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ============================= ======================= ========== =============== ============= ==========================================
+  Group                         Dataset                 Type       Shape           Requirement   Description
+  ============================= ======================= ========== =============== ============= ==========================================
+  /electrodes/{electrodename}   layer                   utf8       1               Optional      Layer of the circuit in which {electrodename} is located
```

Comment:
how does this differ from `region`? Not all regions have layers

Reply:
`layer` is only relevant for the cortex. As per the latest version of the spec, if the electrode is in a region without cortical layers, then `layer` should be"NA". If the electrode is outside the brain, then is should be "Outside"